CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 3.19)
-project(mydb)
+project(DATALINE)
 
 set(CMAKE_CXX_STANDARD 17)
 
-add_executable(mydb src/main.cpp)
+include_directories(src)
+
+add_subdirectory(lib/googletest)  # GTEST_DIR
+
+add_executable(DATALINE src/main.cpp)
```

Comment:
here update to link all the .cpp and/or .h files

Reply:
> here update to link all the .cpp and/or .h files

Already updated.

Reply:
> here update to link all the .cpp and/or .h files

Already updated.